CMakeLists.txt

```diff
@@ -937,10 +1076,12 @@ endif()
 
 # DH and ECC need bigint
 if(NOT WOLFSSL_ASN AND
-   NOT WOLFSSL_DH AND
-   NOT WOLFSSL_ECC AND
-   NOT WOLFSSL_RSA)
-    override_cache(WOLFSSL_FAST_MATH "no")
+    NOT WOLFSSL_DH AND
```

Comment:
Align the NOTs, please (remove whitespace change). Also, while you're here, could you add `AND NOT WOLFSSL_DSA`? Looks like that's missing.